### PR TITLE
[linux] fix using CDVDVideoCodecDRMPRIME on wayland

### PR DESCRIPTION
**Pull request by lrusak**
This fixes an issues where using `CDVDVideoCodecDRMPRIME` is broken with hardware decoding after https://github.com/xbmc/xbmc/commit/8d9b456576f0622a221bc315c56b02005843f190

The `dynamic_cast` to `CWinSystemGbm` would fail and return false when running on wayland. This would only occur if building for GBM and Wayland simultaneously.

### CODE REVIEW

**Review by yol — APPROVED** [review]
I'd put the logic to get the used render node into CWinSystemGbm though, it doesn't really belong in the video codec

**Comment by lrusak** [comment]
> I'd put the logic to get the used render node into CWinSystemGbm though, it doesn't really belong in the video codec

Good point, I've added the change :+1:

File: xbmc/windowing/gbm/drm/DRMUtils.h [hunk 1]
```diff
@@ -89,6 +90,7 @@ class CDRMUtils
   void PrintDrmDeviceInfo(drmDevicePtr device);
 
   int m_renderFd;
+  const char* m_renderDevicePath;
```
Comment (yol) [inline xbmc/windowing/gbm/drm/DRMUtils.h:93]:
```suggestion
  const char* m_renderDevicePath{};
```

better not to have it completely uninitialized

Comment (lrusak) [reply xbmc/windowing/gbm/drm/DRMUtils.h]: thanks 😄

**Review by yol — APPROVED** [review]

**Comment by lrusak** [comment]
jenkins build this please

**Pull request merged**